tests/testapps/CTCatalog/CTCatalog-WinStore10.sln

```diff
@@ -0,0 +1,43 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Thanks VS. It opened as soon as AJ showed up at my office

---
In reply to: [74621157](https://github.com/Microsoft/WinObjC/pull/873#discussion_r74621157) [](ancestors = 74621157)